libs/template/renderer_test.go

```diff
@@ -91,6 +91,7 @@ func assertBuiltinTemplateValid(t *testing.T, template string, settings map[stri
 	})
 
 	b.Tagging = tags.ForCloud(w.Config)
+	b.SetWorkpaceClient(w)
```

Comment:
Test was hanging with a fork bomb because it was using a real client instead of mocked one. Not sure why it started to happen with a new SDK though